src/index.ts

```diff
@@ -76,6 +79,7 @@ const COMMANDS: { [key: string]: [(...args) => void, string, string] } = {
       this.echo(`Pool:    ${CONTRACT_ADDRESS}`);
       this.echo(`Token:   ${TOKEN_ADDRESS}`);
       this.echo(`Minter:  ${MINTER_ADDRESS}`);
+      this.echo(`Prover:  ${DELEGATED_PROVER_URL}`)
```

Comment:
```suggestion
      this.echo(`Prover:  ${DELEGATED_PROVER_URL ?? 'Local' }`)
```

Otherwise we have a blank field in case of prover not set

Reply:
Also it's need to process a void string case

Reply:
I thought that this command only prints envs values, so if env is not set it is ok to print an empty string. Maybe `Prover` isn't the best name although.